configs/11.0/packages/binutils/sources

```diff
@@ -42,3 +42,20 @@ ATSRC_PACKAGE_PATCHES=
 ATSRC_PACKAGE_TARS=
 ATSRC_PACKAGE_DISTRIB=yes
 ATSRC_PACKAGE_BUNDLE=main_toolchain
+
+atsrc_get_patches ()
+{
+	at_get_patch_and_trim \
+		https://sourceware.org/ml/binutils/2017-02/msg00269.html \
+		fix-incorrect-indirection.patch 14 \
+		80e3c4062604f28f186289db0167b592 || return ${?}
+}
+
+atsrc_apply_patches ()
+{
+	#Fix error: comparison between pointer and zero character constant
+	#[-Werror=pointer-compare] in stabs.c
+	patch -p1 < fix-incorrect-indirection.patch
```

Comment:
@tuliom like ` patch -p1 < fix-incorrect-indirection.patch || return ${?}`? What about others places like 8.0? We should fix?